app/router.js

```diff
@@ -7,7 +7,8 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('provider', {path: 'preprints/:slug'}, function () {
+  this.route('provider', {path: '/:provider_id'}, function () {
```

Comment:
This still needs to be `preprints/:provider_id`, because reasons @laurenbarker can explain (so we can have a different namespace for non-preprint providers without changing URLs, I think?)